lib/profiles.js

```diff
@@ -16,13 +16,18 @@ const KMS_MODULE = 'ssm-v1';
  *
  *
  * @param {string} accountId - Id of the account to associate with the Profile.
- * @param {string} [didMethod] - Supported: 'key' and 'v1'.
+ * @param {string} didMethod - Supported: 'key' and 'v1'.
  * @param {object} [didOptions] - Hashmap of optional DID method options.
  *
  * @return {Promise<Profile>} resolves to a profile's settings.
  */
 module.exports.create = async ({accountId, didMethod, didOptions} = {}) => {
   assert.string(accountId, 'accountId');
+  assert.string(didMethod, 'didMethod');
+  assert.optionalObject(didOptions, 'didOptions');
+  if(didMethod !== 'key' && didMethod !== 'v1') {
```

Comment:
Let's opt for creating an array named `SUPPORTED_DID_METHODS`. We can place our supported methods `key` and `v1` in that array. Then, this check would simplify to checking if `didMethod` is included in the array.

A similar pattern can be found here: https://github.com/digitalbazaar/bedrock-web-profile-manager/blob/master/ProfileManager.js#L77-L79

Reply:
Updated.